### PR TITLE
#858: Change text when downloading Bitcoin to include version number

### PR DESCRIPTION
**Pull request by acarlton**
Simple text change to add bitcoind version number to download wording as per #858. First issue exploration.

### CODE REVIEW

**Comment by pythcoiner** [comment]
ACK  d2ef685595dc393b148ed27c16311d302a11a9d1 (tested on signet)

image

File: gui/src/installer/view.rs [hunk 1]
```diff
@@ -1043,7 +1044,9 @@ pub fn start_internal_bitcoind<'a>(
                     DownloadState::Downloading { progress } => Row::new()
                         .spacing(10)
                         .align_items(Alignment::Center)
-                        .push(text(format!("Downloading... {progress:.2}%"))),
+                        .push(text(format!(
+                            "Downloading Bitcoin {version}... {progress:.2}%"
```
Comment (jp1ac4) [inline gui/src/installer/view.rs:1048]: Could you please change it to "Bitcoin Core".

**Review by jp1ac4 — CHANGES_REQUESTED** [review]
Thanks for this PR.

There's a small change to the wording needed. You can squash your commits already when making that change. Thanks.

**Review by jp1ac4 — APPROVED** [review]
ACK https://github.com/wizardsardine/liana/commit/b4f505c4312c3ef4c54dde82a940e923c2b3ca1d.

image

**Comment by pythcoiner** [comment]
ACK b4f505c4312c3ef4c54dde82a940e923c2b3ca1d

**Comment by darosior** [comment]
Thanks!

**Pull request merged**